### PR TITLE
User story 1

### PR DESCRIPTION
**Pull request by stschuessler**
Finished first user story. Please have a look :)

### CODE REVIEW

**Review by mbosselmann — APPROVED** [review]
Looks good! :-)

File: client/src/components/Card.spec.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { render, screen } from '@testing-library/react'
+import Card from './Card'
+
+describe('Card', () => {
+  const text = 'This is a test text'
+  const author = 'This is a test author'
+
+  it('renders a question', () => {
+    render(<Card text={text} author={author} />)
+  })
```
Comment (macfrei) [inline client/src/components/Card.spec.js:10]: This doesn't test anything, you can delete this.

File: client/src/components/Card.spec.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+  it('it has two paragraph tags', () => {
+    render(<Card text={text} author={author} />)
+
+    const paragraph1 = screen.getByText('This is a test text')
+    const paragraph2 = screen.getByText('This is a test author')
```
Comment (macfrei) [inline client/src/components/Card.spec.js:16]: Maybe name it `textParagraph` and `authorParagraph`

File: client/src/components/Main.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components'
+import Card from './Card';
+
+export default function Main({data}){
+	return (
+	<StyledMain>
+		{data.map((question) => (
+			<Card text={question.text} author={question.author} />
```
Comment (macfrei) [inline client/src/components/Main.js:8]: `key` is missing! 😱 
```
<Card text={question.text} author={question.author} key={question.id} />
```

File: client/src/components/Main.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+		))}
+	</StyledMain>
+	)
+}
+
+const StyledMain = styled.main`
+  main {
+    padding: 20px; 
+	white-space: wrap;
+  }
```
Comment (macfrei) [inline client/src/components/Main.js:18]:
```
        padding: 20px; 
	white-space: wrap;
```

File: client/src/index.js [hunk 5]
```diff
@@ -1,17 +1,19 @@
...
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import React from 'react'
+import ReactDOM from 'react-dom'
+import App from './App'
+import reportWebVitals from './reportWebVitals'
+import GlobalStyles from './GlobalStyles'
+import { useState, useEffect } from 'react'
```
Comment (macfrei) [inline client/src/index.js:6]: You don't need this here.

**Review by macfrei — COMMENTED** [review]
👍

**Pull request merged**